restler/engine/bug_bucketing.py

```diff
@@ -147,6 +154,41 @@ def _get_bug_hash(self, origin, sequence, hash_full_request, additional_str):
 
         return f'{origin}_{str_to_hex_def(request_str)}'
 
+    def _attempt_reproduce(self, reproduce, sequence, bug_code, bucket):
+        """ Attempts to reproduce a bug, if requested
+
+        @param reproduce: If False, do not attempt to reproduce
+        @type  reproduce: Bool
+        @param sequence: The bug sequence to replay
+        @type  sequence: Sequence
+        @param bug_code: The status code that was received when the bug was identified.
+        @type  bug_code: Str
+        @param bucket: Pointer to the bug bucket related to this bug
+        @type  bucket: OrderedDict - {seq_hex: BugBucket}
+        @return: Tuple containing whether the bug was reproduced,
+                 the new retry count, and the new reproduced count
+        @rtype : Tuple(bool, int, int)
+
+        """
+        if reproduce:
```

Comment:
>reproduce [](start = 11, length = 9)

This looks strange.  Suggest moving this out of the function, or making the reproduce parameter an integer, and handling >0 vs. 0 in the else.